src/snowflake/connector/cursor.py

```diff
@@ -786,6 +791,7 @@ def execute_async(self, *args, **kwargs):
         for documentation.
         """
         kwargs["_exec_async"] = True
+        kwargs["_no_retry"] = kwargs.pop("no_retry", False)
```

Comment:
This breaks the documentation `This function takes the same arguments as execute`. This function should accept `_no_retry` like `execute`.
```suggestion
```